CMakeLists.txt

```diff
@@ -2,37 +2,49 @@ cmake_minimum_required(VERSION 3.5.1)
 project(CMaizeSuperbuild VERSION 1.0.0 LANGUAGES CXX)
 
 # This is where we will install artifacts during the build stage
-set(stage_directory "${CMAKE_BINARY_DIR}/stage/${CMAKE_INSTALL_PREFIX}")
+set(stage_directory "${CMAKE_BINARY_DIR}/stage${CMAKE_INSTALL_PREFIX}")
 
 # Tell compiler to not include debug flags
 set(CMAKE_BUILD_TYPE_INIT "Release")
-
-# Add cmake config location to paths to search
-list(APPEND CMAKE_PREFIX_PATH ${stage_directory})
+set(EXTERNAL_DIR "helloworld/cmake/external")
```

Comment:
```suggestion
set(EXTERNAL_DIR "helloworld/external")
```
if we're going to actually put the external includes in `includes` and libraries in `libs` I think I like it better without the `cmake` prefix. When I recommended the `cmake` prefix I thought we were going to have to put everything in one directory. We can give this a shot, but I'm not too sure how widespread the use of `CMAKE_INSTALL_X` is with the exception of `X` equals `PREFIX`.